nautobot_golden_config/templates/nautobot_golden_config/configplan_create.html

```diff
@@ -6,136 +6,136 @@
 {% block title %}Generate Config Plans{% endblock %}
 
 {% block form %}
-  <div class="panel panel-default">
-  <div class="panel-heading"><strong>Plan Details</strong></div>
-  <div class="panel-body">
-    {% render_field form.plan_type %}
-    {% render_field form.change_control_id %}
-    {% render_field form.change_control_url %}
-    {% render_field form.feature %}
-    {% render_field form.commands %}
-  </div>
-  <div class="panel-heading"><strong>Plan Filters</strong></div>
-  <div class="panel-body">
-    <span class="help-block">
-      <p>Note: Selecting no filters will generate plans for all applicable devices.</p>
-    </span>
-    {% render_field form.tenant_group %}
-    {% render_field form.tenant %}
-    {% render_field form.location %}
-    {% render_field form.rack_group %}
-    {% render_field form.rack %}
-    {% render_field form.role %}
-    {% render_field form.manufacturer %}
-    {% render_field form.platform %}
-    {% render_field form.device_type %}
-    {% render_field form.device %}
-    {% render_field form.tags %}
-    {% render_field form.status %}
-  </div>
-</div>
+    <div class="card">
+        <div class="card-header"><strong>Plan Details</strong></div>
+        <div class="card-body">
+            {% render_field form.plan_type %}
+            {% render_field form.change_control_id %}
+            {% render_field form.change_control_url %}
+            {% render_field form.feature %}
+            {% render_field form.commands %}
+        </div>
+        <div class="card-header"><strong>Plan Filters</strong></div>
+        <div class="card-body">
+            <span class="help-block">
+                <p>Note: Selecting no filters will generate plans for all applicable devices.</p>
+            </span>
+            {% render_field form.tenant_group %}
+            {% render_field form.tenant %}
+            {% render_field form.location %}
+            {% render_field form.rack_group %}
+            {% render_field form.rack %}
+            {% render_field form.role %}
+            {% render_field form.manufacturer %}
+            {% render_field form.platform %}
+            {% render_field form.device_type %}
+            {% render_field form.device %}
+            {% render_field form.tags %}
+            {% render_field form.status %}
+        </div>
+    </div>
 
 {% endblock %}
 
 {% block buttons %}
-{% include "nautobot_golden_config/job_result_modal.html" with modal_title="Generate Config Plans" %}
+    {% include "nautobot_golden_config/job_result_modal.html" with modal_title="Generate Config Plans" %}
 
-<a href="#" id="startJob" class="openBtn" data-backdrop="static">
-  <button type="submit" class="btn btn-primary">Generate</button>
-</a>
+    <a class="openBtn" data-backdrop="static" href="#" id="startJob">
```

Comment:
no `bs-` needed on the data-backdrop?

Reply:
Just confirmed this is the only one missing. Thanks for catching that.